Sources/PromotedAIMetricsSDK/MetricsLogger.swift

```diff
@@ -114,10 +164,15 @@ public class MetricsLogger: NSObject {
     self.store = store
     self.logMessages = []
     self.userID = nil
-    self.logUserID = nil
-    self.sessionID = nil
+    self.logUserIDProducer = IDProducer(initialValueProducer: {
+      return store.logUserID ?? idMap.logUserID()
+    }, nextValueProducer: {
+      return idMap.logUserID()
+    })
+    self.sessionIDProducer = IDProducer { return idMap.sessionID() }
+    self.viewIDProducer = IDProducer { return idMap.viewID() }
```

Comment:
is this going to reuse the last view id on relaunch?  does reloading of the app cause the last view id to be used in that case?  i worry that the last view before restart and the first view after aren't the same view.

Reply:
No, this won't re-use view ID when restarting the app. We regenerate view IDs and session IDs each time.